politeiawww/api/www/v1/v1.go

```diff
@@ -1360,3 +1362,13 @@ type WSSubscribe struct {
 type WSPing struct {
 	Timestamp int64 `json:"timestamp"` // Server side timestamp
 }
+
+// MessageProposer is a request that allows administrators to 'ping' or
+// message a proposer in lieu of simply authorizing or censoring a proposal.
+type MessageProposer struct {
+	Token   string `json:"token"`   // Token of the proposal that is receiving a message.
```

Comment:
I think we should make this a more generic route (remove the token requirement) that allows the admin to send the user a generic message.  This will allow us to add more use cases to it in the future if need be. We can handle the message template for specific use cases on the gui side.

For the email template on the server side, it would be something like:
```
You have received a message from administrator {{.AdminUsername}}:

{{.Message}}
```

Then the gui would have a message template for specific use cases like telling a proposal author they're about to abandon their proposal.